### PR TITLE
ci: Add more kernel tests

### PR DESCRIPTION
**Pull request by jschwinger233**
Covers IPv4 + IPv6 for TCP + UDP under WAN + LAN.

IPv6 UDP is buggy so I skipped that: #102 #387

### CODE REVIEW

File: .github/workflows/kernel-test.yml [hunk 1]
```diff
@@ -135,30 +157,111 @@ jobs:
             }
 
             routing {
-                dip(1.1.1.1) -> proxy
-                fallback: direct
+                pname(dae) -> direct
```
Comment (mzz2017) [inline .github/workflows/kernel-test.yml:160]: Why we need this?

Comment (jschwinger233) [reply .github/workflows/kernel-test.yml:160]: https://github.com/daeuniverse/dae/issues/384

Pid matching doesnt work inside container

**Comment by jschwinger233** [comment]
IPv6 keeps failing, in my opinion this is because github action doesn't IPv6 outbound to world: https://github.com/actions/runner-images/issues/668.

I therefore plan to test IPv6 3-way handshake only, to make sure packets originated by a local process can finally arrive at dae process.

As for IPv6 UDP, it seems to be buggy on main. Will troubleshot that later.

**Comment by jschwinger233** [comment]
好耶，IPv6 TCP for WAN / LAN 也测过了，ready for review.

**Comment by mzz2017** [comment]
@jschwinger233 Exciting news. What about ipv6 udp?

**Comment by mzz2017** [comment]
@jschwinger233 I think we should also test it whatever it has bug or not.

**Comment by jschwinger233** [comment]
@mzz2017 IPv6 UDP testcase added.

**Comment by mzz2017** [comment]
Cool. Thanks for your brilliant work. I think it is ready to merge.

**Review by mzz2017 — DISMISSED** [review]

**Pull request merged**